spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+# rubocop:disable Metrics/BlockLength
```

Comment:
- [ ] Rubocop is a static code analyzer and formatted. Let's kindly fix the issue it reports instead of disabling it.